dev_environment/config.yaml

```diff
@@ -7,3 +7,4 @@ configs:
     host_l3af_debug_port: '38899'
     host_l3afd_code_dir: '/code/l3afd'
     traffic_mirroring: 'false'
+    host_l3afd_jammy: 'true'
```

Comment:
I would recommend making this generic, so we don't have to change the name of this every release.  Maybe call it something like host_distro